SDKBuildScripts/ue4_CPP_build.bat

```diff
@@ -1,24 +1,24 @@
 rem === Cleaning existing files from UnrealCppSdk ===
 pushd ..\..\sdks\UnrealCppSdk
-pushd PlayFabSDK\Plugins\PlayFab\Source\PlayFabProxy\Private\Proxy
+pushd PlayFabSDK\Plugins\PlayFabProxy\Source\PlayFabProxy\Private\Proxy
```

Comment:
What does the proxy do, and why move it?

Reply:
Proxy, as provided in the previous UnrealCPP SDK, implements some BP functions, however wasn't feature complete in the previous version. I moved it to its own plugin so people can throw it in plugins if wanted. Technically the proxy could be removed as well, I'm just providing the same things as before.

Reply:
OIC, ok.  So the proxy would be the theoretical blueprints tie-in.  NICE. That seems like a good plan for a theoretical future where we have a consistent plugin/set-of-plugins that supports both CPP and Blueprint.

Reply:
Well, again, it was already provided before. It's still not feature complete but I plan to continue to edit it, flush it out more.